ui/lib/replication/addon/templates/components/replication-summary.hbs

```diff
@@ -0,0 +1,349 @@
+                CA path <em class="is-optional">(optional)</em>
+              </label>
+              <div class="control">
+                {{input value=ca_path id="ca_path" name="ca_file" class="input"}}
+              </div>
+              <p class="help has-text-grey">
+                Specifies the path to a CA root directory containing PEM-format files that the secondary can use when unwrapping the token from the primary.
+              </p>
+            </div>
+            <p>
+              Note: If both <code>CA file</code> and <code>CA path</code> are not given, they default to system CA roots.
+            </p>
+          {{/if}}
+        {{else}}
+          <p>The token you are using is not authorized to enable secondary replication.</p>
+        {{/if}}
+      {{/if}}
+    </div>
+    {{#if (or (and (eq mode 'primary') cluster.canEnablePrimary) (and (eq mode 'secondary') cluster.canEnableSecondary))}}
+      <div class="field is-grouped box is-fullwidth is-bottomless">
+        <div class="control">
+          <button type="submit" class="button is-primary" disabled={{disallowEnable}} data-test-replication-enable=true>
+            Enable Replication
+          </button>
+        </div>
+      </div>
+    {{/if}}
+  </form>
+{{else if showModeSummary}}
+  <PageHeader as |p|>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        Replication
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <h3 class="title is-flex-center is-5 is-marginless">
+      {{!-- TODO convert to a new component that doesn't use partials
+      {{i-con class="has-text-grey is-medium" glyph="replication" size=20}}
+      --}}
+      Disaster Recovery (DR)
+    </h3>
+    {{#if cluster.dr.replicationEnabled}}
+      {{#link-to
+        "mode.index"
+        "dr"
+        class="link-plain"
+      }}
+        {{replication-mode-summary
+          mode="dr"
+          cluster=cluster
+          tagName="span"
+        }}
+      {{/link-to}}
+    {{else}}
+      {{replication-mode-summary
+        mode="dr"
+        cluster=cluster
+        tagName="div"
+      }}
+    {{/if}}
+  </div>
+  <div class="box is-bottomless is-fullwidth is-marginless">
+    <h3 class="title is-flex-center is-5 is-marginless">
+      {{!-- TODO convert to a new component that doesn't use partials
+      {{i-con class="has-text-grey is-medium" glyph="perf-replication" size=20}}
+      --}}
+      Performance
+    </h3>
+    {{#if cluster.dr.replicationEnabled}}
+      {{#link-to
+        "mode.index"
+        "performance"
+        class="link-plain"
+      }}
+        {{replication-mode-summary
+          mode="performance"
+          cluster=cluster
+          tagName="span"
+        }}
+      {{/link-to}}
+    {{else}}
+      {{replication-mode-summary
+        mode="performance"
+        cluster=cluster
+        tagName="div"
+      }}
+    {{/if}}
+  </div>
+{{else}}
```

Comment:
😳 oh golly, this template is also quite large. is it possible to break it up?

Reply:
😬 this is the one I tried to break up it wasn't super straightforward. I'm gonna try doing the summary mode one, but think this one might be hard to break up without a larger refactor.